pkg/machine/ignition.go

```diff
@@ -495,24 +499,13 @@ Delegate=memory pids cpu io
 		if _, err := os.Stat(sslCertFile); err == nil {
 			certFiles = getCerts(sslCertFile, false)
 			files = append(files, certFiles...)
+		}
+	}
 
-			if len(certFiles) > 0 {
-				setSSLCertFile := fmt.Sprintf("export %s=%s", "SSL_CERT_FILE", filepath.Join("/etc/containers/certs.d", filepath.Base(sslCertFile)))
-				files = append(files, File{
-					Node: Node{
-						Group: getNodeGrp("root"),
-						Path:  "/etc/profile.d/ssl_cert_file.sh",
-						User:  getNodeUsr("root"),
-					},
-					FileEmbedded1: FileEmbedded1{
-						Append: nil,
-						Contents: Resource{
-							Source: encodeDataURLPtr(setSSLCertFile),
-						},
-						Mode: intToPtr(0644),
-					},
-				})
-			}
+	if sslCertDir, ok := os.LookupEnv("SSL_CERT_DIR"); ok {
+		if _, err := os.Stat(sslCertDir); err == nil {
+			certFiles = getCerts(sslCertDir, true)
+			files = append(files, certFiles...)
```

Comment:
On the fence, any thoughts on at least a debug error or perhaps a warning message here if the certdir can't be opened?

Reply:
Like so?
![image](https://user-images.githubusercontent.com/411913/202658816-c3504a32-0dc9-4ed7-b8f0-d0a0ddcc8fd0.png)

Reply:
LGTM

Reply:
Yes, TY!

Reply:
@bjorndown Are you going to add this patch?

Reply:
@rhatdan It is included in the latest commit.